addons/addon-base-raas/packages/base-raas-services/lib/aws-accounts/__tests__/aws-cfn-service.test.js

```diff
@@ -26,9 +26,6 @@ const AuthServiceMock = require('@amzn/base-services/lib/authorization/authoriza
 jest.mock('@amzn/base-services/lib/audit/audit-writer-service');
 const AuditServiceMock = require('@amzn/base-services/lib/audit/audit-writer-service');
 
-jest.mock('@amzn/base-services/lib/settings/env-settings-service');
-const SettingsServiceMock = require('@amzn/base-services/lib/settings/env-settings-service');
```

Comment:
looks like you are using different mocking data for different test cases?

is it possible to register the mocking `SettingService` in the targeted test cases?

Reply:
Yes, there are two ways to implementation to mock settings. 
Recoverd to SettingsServiceMock way.